### DISCUSSION TITLE
aiohttp_swagger3 AND  aiohttp_cors

### DISCUSSION THREAD

**Discussion by RVC69** [hh-h/aiohttp-swagger3#96] [Q&A] [answered]
Hello,
I cannot figure out how to use both of these libraries : aiohttp_cors + aiohttp_swagger
Using cors : 
         resource = cors.add(app.router.add_resource(url))
         cors.add(resource.add_route(method, handler)

but if you are trying to use swagger just after:
        swagger.add_route(method, route, handler)

You will get an error : 
File "d:\programs\python\3.9.7_64bits\lib\site-packages\aiohttp\web_urldispatcher.py", line 346, in add_route
    raise RuntimeError(
RuntimeError: Added route will never be executed, method XXX is already registered

So any idea how to add swagger support to CORS enabled resources ?
regards

**Comment by hh-h** [comment]
Hello! Please, provide an example so I can play with it, thank you.

**Comment by RVC69** [comment, marked as answer]
Hello !
I am just stupid.
You have just to add cors to swagger route or resource AND NOT trying to add swagger to cors 
regards
